DEPLOYMENT.md

```diff
@@ -0,0 +1,57 @@
+# Configuring RHMAP Core to use non-wildcard SSL routing
+
+For a standard RHMAP installation deployed on top of OpenShift, we recommend that a wild card SSL certificate be installed on the OpenShift router. However, in situations where this is not desirable, RHMAP can be exposed using single URL.
+
+## RHMAP Proxy
+
+RHMAP Proxy is a separate component which should be deployed to OpenShift per MBaaS project. Its purpose is to route requests that are not using the standard URL scheme. The RHMAP Proxy application is bundled within the RHMAP RPM in the form of an OpenShift template. Creating this application will create an OpenShift route which will expose this service. Carry out the following steps to configure RHMAP Core to use non-wildcard routing.
+
+**_1.  Setting Millicore environment variable for client side connectivity_**
+
+Upon initialization, a client application must call out to RHMAP Core to discover the URL to which it should make API requests. We must set the following environment variable in Millicore to allow this to work behind RHMAP Proxy.
+
+```
+oc env dc millicore -n rhmap-core EXTERNAL_HOST=https://rhmaprouter.internal.domain.com
+```
+
+NOTE: The `EXTERNAL_HOST` value must match the value of the route which you will pass to the template for RHMAP Proxy
+
+
+**_2. Configure an MBaaS to use RHMAP Proxy_**
+
+When creating an MBaaS target, you must specify an external URL via the `”External MBaaS Host”` form field. The value provided here when creating the MBaaS target must match the value of `EXTERNAL_HOST` set previously.
```

Comment:
I believe we moved away from this change did we not? The reason was that I think @wtrocki said they are not necessarily the same but open to correction on this?